src/components/NavBar/index.tsx

```diff
@@ -1,28 +1,20 @@
 import React from 'react'
-import { LinkContainer } from 'react-router-bootstrap'
-import { Navbar, Nav } from 'react-bootstrap'
+import { Nav, Navbar } from 'react-bootstrap'
+import LogOut from '../LogOut'
 
 const NavBar = () => {
+  const isAuthenticated = true // temporarily included. Will be after confirmation of authentication from backend team.
```

Comment:
This should be stored inside `redux`. Perhaps you can make a `login` state that has all the related information about credential stuffs